src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -23,24 +23,32 @@ public ActivityAttribute ()
 #endif
 		public bool                   AllowTaskReparenting    {get; set;}
 		public bool                   AlwaysRetainTaskState   {get; set;}
+		public bool                   AutoRemoveFromRecents   {get; set;}
+		public string                 Banner                  {get; set;}
 		public bool                   ClearTaskOnLaunch       {get; set;}
 #if ANDROID_26
 		public string                 ColorMode               {get; set;}
 #endif
 		public ConfigChanges          ConfigurationChanges    {get; set;}
+		public string                 Description             {get; set;}
 #if ANDROID_24
 		public bool                   DirectBootAware         {get; set;}
 #endif
+		public DocumentLaunchMode     DocumentLaunchMode      {get; set;}
+		public string                 EnableVrMode            {get; set;}
```

Comment:
This is not documented at: https://developer.android.com/guide/topics/manifest/activity-element

Reply:
@jonpryor while `EnableVrMode` isn't document there, it is here: [https://developers.google.com/vr/reference/vr-manifest](https://developers.google.com/vr/reference/vr-manifest)